staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -866,3 +872,11 @@ type SerializedNodeConfigSource struct {
 	// +optional
 	Source v1.NodeConfigSource `json:"source,omitempty" protobuf:"bytes,1,opt,name=source"`
 }
+
+// LoggingConfig contains logging options
+type LoggingConfig struct {
+	// LoggingFormat Flag specifies the structure of log messages.
+	// default value of loggingFormat is `text`
+	// Refer [Logs Options](https://github.com/kubernetes/component-base/blob/master/logs/options.go) for more information.
+	LoggingFormat string `json:"loggingFormat,omitempty"`
```

Comment:
this is already in a logging options type, can name this `format`:
```suggestion
	Format string `json:"format,omitempty"`
```